share/container-build.sh

```diff
@@ -2,17 +2,13 @@
 
 #
 # SPDX-FileCopyrightText:  2023, Iker Pedrosa <ipedrosa@redhat.com>
+# SPDX-FileCopyrightText:  2024, Iker Pedrosa <ipedrosa@redhat.com>
 #
 # SPDX-License-Identifier:  BSD-3-Clause
 #
 
-for FILE in share/containers/*; do
-	IFS='/'
-	read -ra ADDR <<< "$FILE"
-	IFS='.'
-	read -ra ADDR <<< "${ADDR[2]}"
-	IFS=''
-	if ! docker build -f $FILE . --output build-out/${ADDR[0]}; then
-		exit
-	fi
-done
+pushd share/ansible/ || exit
+ansible-playbook playbook.yml -i inventory.ini -e 'distribution=alpine'
+ansible-playbook playbook.yml -i inventory.ini -e 'distribution=debian'
+ansible-playbook playbook.yml -i inventory.ini -e 'distribution=fedora'
+popd || exit
```

Comment:
Nice!  I didn't know pushd(1) and popd(1) existed.  :)

Reply:
I discovered them two months ago or so. I don't think they are very popular

Reply:
I was wondering the other day if there would be a way to keep more than one level of directory stack.  `cd -` is useful, but when you need to jump back and forth with more than two dirs, it doesn't help.  pushd(1) is what I was looking for.  :)